freqtrade/tests/testdata/download_backtest_data.py

```diff
@@ -28,7 +28,7 @@
 print('About to download pairs:', PAIRS)
 
 # Init Bittrex exchange
-exchange._API = Bittrex({'key': '', 'secret': ''})
+exchange._API = ccxt.bittrex({'key': '', 'secret': ''})
```

Comment:
shouldn't we call the `exchange.init` rather than init specific exchange?

Reply:
Yeah probably. This particular file is not connected to the bot in any way and can be used for manual downloading. So it might be reasonable not to link it to the config but rather specify the exchange via command line